src/uwtools/api/config.py

```diff
@@ -246,3 +246,20 @@ def validate(
 """.format(
     extensions=", ".join(_FORMAT.extensions())
 ).strip()
+
+__all__ = [
+    "FieldTableConfig",
+    "INIConfig",
+    "INIConfig",
+    "SHConfig",
+    "YAMLConfig",
+    "compare",
+    "get_fieldtable_config",
+    "get_ini_config",
+    "get_nml_config",
+    "get_sh_config",
+    "get_yaml_config",
+    "realize",
+    "realize_to_dict",
+    "validate",
+]
```

Comment:
This is necessary to instruct Sphinx to produce docs for imported classes as well as things defined in this module.